.github/workflows/ci.yaml

```diff
@@ -39,7 +44,11 @@ jobs:
           cp ~/.m2/settings.xml settings.xml
           docker build -t management-api-for-dse-builder -f ./Dockerfile-build-dse ./
           docker tag management-api-for-dse-builder management-api-for-apache-cassandra-builder
-          docker build -t mgmtapi-3_11 -f Dockerfile-3_11 .
+          docker buildx build --load \
+            --tag mgmtapi-3_11 \
+            --file Dockerfile-oss \
+            --target oss311 \
+            --platform linux/amd64 .
```

Comment:
Only building here for "linux/amd64" as a multiarch image cannot be loaded into the local docker images. This should be good enough for running the integration tests.